From 94fe5e052f6fa77acca513b6c6811fb8e1f1672b Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 18:58:17 -0400
Subject: [fs] kobject: export kobj_sysfs_ops

Message-id: <1407179109-13714-11-git-send-email-sandeen@redhat.com>
Patchwork-id: 87183
O-Subject: [RHEL7.1 PATCH 010/422] kobject: export kobj_sysfs_ops
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Jeff Mahoney <jeffm@suse.com>

commit 29dfe2dc0e8f85c5656d13bb4c78a5ffca54c452
Author: Jeff Mahoney <jeffm@suse.com>
Date:   Fri Nov 1 13:06:56 2013 -0400

    kobject: export kobj_sysfs_ops

    struct kobj_attribute implements the baseline attribute functionality
    that can be used all over the place. We should export the ops associated
    with it.

    Signed-off-by: Jeff Mahoney <jeffm@suse.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
    Signed-off-by: Josef Bacik <jbacik@fusionio.com>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/lib/kobject.c b/lib/kobject.c
index b7e29a6..abdc55e 100644
--- a/lib/kobject.c
+++ b/lib/kobject.c
@@ -709,6 +709,7 @@ const struct sysfs_ops kobj_sysfs_ops = {
  .show = kobj_attr_show,
  .store = kobj_attr_store,
 };
+EXPORT_SYMBOL_GPL(kobj_sysfs_ops);
 
 /**
  * kset_register - initialize and add a kset.
-- 
1.7.1